.cspell.json

```diff
@@ -2,11 +2,14 @@
   "version": "0.2",
   "language": "en",
   "words": [
+    "accountdefinitelydoesnotexistwiththisprefix",
```

Comment:
Extremely nitpicky nitpick: this and `nosuchusername` we should probably just `cSpell: ignore` instead of adding to the dictionary

Reply:
👍